### ISSUE TITLE
size format option 'kb' actually returns value in Kib 

### ISSUE THREAD

**Issue by sushidub** [wp-cli/db-command#102]
I'm sure nobody notices or cares, but it did actually throw me for a loop trying to chase down a separate db related issue.

The CLI docs for **db** say _**size_format**_ flag can return optional values as follows:
```bash
$ wp help db size
  ...
  [--size_format]
    Display the database size only, as a bare number.
    ---
    default: b
    options:
     - b (bytes)
     - kb (kilobytes)
     - mb (megabytes)
     - gb (gigabytes)
     - tb (terabytes)
     ---
```
Goes on to show an example of returned values for options =b, =kb, =mb where the kb value actually seems to be reporting in kibibytes

**Solution:** Add footnote or short blurb in the doc mentioning the returned value for kb will report what might be slightly lower than expected since its factored by 1024 (not 1000)

```bash
1 KiB (Kibibyte) = 1,024 B (Bytes) (2^10 Bytes)
1 kb  (Kilobit)  =   125 B (Bytes) (10^3 Bits ÷ (8 bits / byte) = 125 B)
1 kB  (Kilobyte) = 1,000 B (Bytes) (10^3 Bytes)
```

**Comment by schlessera** [comment]
@sushidub You are right, the current usage of the sizes is technically wrong.

As we currently use a lowercase entry for shell convenience, I wonder if we shouldn't just extend the amount of available options and let people choose what they prefer:
```
$ wp help db size
  ...
  [--size_format]
    Display the database size only, as a bare number.
    ---
    default: b
    options:
     - b   (Legacy Byte setting, with no conversion)
     - kb  (Legacy Kilobyte setting, with 1 kb = 1,024 b)
     - mb  (Legacy Megabyte setting, with 1 mb = 1,024 kb)
     - tb  (Legacy Terabyte setting, with 1 tb = 1,024 gb)
     - gb  (Legacy Gigabyte setting, with 1 gb = 1,024 mb)
     - B   (ISO Byte setting, with no conversion)
     - kB  (ISO Kilobyte setting, with 1 kB  = 1,000 B)
     - KiB (ISO Kibibyte setting, with 1 KiB = 1,024 B)
     - MB  (ISO Megabyte setting, with 1 MB  = 1,000 kB)
     - MiB (ISO Mebibyte setting, with 1 MiB = 1,024 KiB)
     - GB  (ISO Gigabyte setting, with 1 GB  = 1,000 MB)
     - GiB (ISO Gibibyte setting, with 1 GiB = 1,024 MiB)
     - TB  (ISO Terabyte setting, with 1 TB  = 1,000 GB)
     - TiB (ISO Tebibyte setting, with 1 TiB = 1,024 GiB)
     ---
```

That might overwhelm people, though...

**Comment by sushidub** [comment]
It would I think. 
For me (altho I'm not sure how I stack up to most cli users), simply having a note in the db help print out that clarifies which setting was used to perform the calculation would be enough to do the math myself using bytes <-- which calculates correctly either way. 
In the short term it may only suffice as a time saver should somebody like myself stumble on it. Long term maybe the divisor should just be 1,000. I'm not sure. 
Thnx tho.

**Comment by schlessera** [comment]
Yes, I agree, that is the better approach for now, seeing the above monstrosity.

So, the solution for now is to add a line to parameter description that calculations are done with the non-ISO multiples of 1024, instead of 1000.

**Comment by schlessera** [comment]
Fixed via #104
